src/pages/buyers-guide/why-we-made.js

```diff
@@ -34,7 +34,7 @@ var BuyersGuide = React.createClass({
                 {this.context.intl.formatMessage({id: 'wwmt_2'})}
               </p>
               <p className="zilla-slab">
-                {this.context.intl.formatMessage({id: 'wwmt_3'})}
+                {this.context.intl.formatMessage({id: 'wwmt_evergreen3'})}
```

Comment:
I don't think this string exists in the why-we-made-this.properties file.

Reply:
You mean this one? https://github.com/mozilla/advocacy.mozilla.org/blob/master/buyers-guide-locales/en-US/why-we-made-this.properties#L14